src/main/java/pl/edu/agh/firecell/model/Material.java

```diff
@@ -0,0 +1,36 @@
+package pl.edu.agh.firecell.model;
+
+public enum Material {
+    // TODO:
+    // consider putting them into constant
```

Comment:
Should we resolve TODOs before merge?

I'm not sure what does "put into constant" mean in this case.

Reply:
For now these hardcoded parameters of materials can stay there, but in future maybe it will be better to store them in static final constants, or even in a file.